From 165f0f012f8d4bebb12d188c1c2a369d233d586e Mon Sep 17 00:00:00 2001
From: Christian Hesse <mail@eworm.de>
Date: Tue, 10 Jan 2017 10:01:54 +0100
Subject: [PATCH 1/1] fix test with freetype2 2.7.1

Upstream freetype2 applied a patch from openSuSE [0]:

[pcf] Enrich family name with foundry name and glyph width info.
3576487add2f0e9691e8c43f8b0cd8e61aebf14f

This prepends the foundry name plus a space to the family name. We needs
change the font name in expexted test output.

[0] http://git.savannah.gnu.org/cgit/freetype/freetype2.git/commit/?id=3576487add2f0e9691e8c43f8b0cd8e61aebf14f

Signed-off-by: Christian Hesse <mail@eworm.de>
---
 test/out.expected | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/test/out.expected b/test/out.expected
index 39634c5..f3a8856 100644
--- a/test/out.expected
+++ b/test/out.expected
@@ -1,8 +1,8 @@
-Fixed:pixelsize=16
-Fixed:pixelsize=6
+Misc Fixed:pixelsize=6
+Sony Fixed:pixelsize=16
 =
-Fixed:pixelsize=16
-Fixed:pixelsize=6
+Misc Fixed:pixelsize=6
+Sony Fixed:pixelsize=16
 =
-Fixed:pixelsize=16
-Fixed:pixelsize=6
+Misc Fixed:pixelsize=6
+Sony Fixed:pixelsize=16
-- 
2.11.0

